deploy/coins/bitcoin/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Configure Environment
+FROM alpine:latest as builder
+
+ENV BITCOIN_VERSION=23.0
+ENV GLIBC_VERSION=2.28-r0
+ENV BITCOIN_DATA=/home/bitcoin/.bitcoin
+
+WORKDIR /home/bitcoin
+
+RUN apk update \
+	&& apk --no-cache add ca-certificates gnupg bash su-exec 
+
+# Get Binary
+RUN wget https://bitcoincore.org/bin/bitcoin-core-${BITCOIN_VERSION}/bitcoin-${BITCOIN_VERSION}-x86_64-linux-gnu.tar.gz \
+	&& wget https://bitcoincore.org/bin/bitcoin-core-${BITCOIN_VERSION}/SHA256SUMS \
+	&& wget https://bitcoincore.org/bin/bitcoin-core-${BITCOIN_VERSION}/SHA256SUMS.asc
+
+# Verify Binary
+# For the PGP keys of builders and developers, refer to 
+# https://github.com/bitcoin/bitcoin/tree/master/contrib/builder-keys 
+ENV KEYS 152812300785C96444D3334D17565732E08E5E41 0AD83877C1F0CD1EE9BD660AD7CC770B81FD22A8 590B7292695AFFA5B672CBB2E13FC145CD3F4304 28F5900B1BB5D1A4B6B6D1A9ED357015286A333D 637DB1E23370F84AFF88CCE03152347D07DA627C CFB16E21C950F67FA95E558F2EEB9F5CC09526C1 F4FC70F07310028424EFC20A8E4256593F177720 D1DBF2C4B96F2DEBF4C16654410108112E7EA81F 287AE4CA1187C68C08B49CB2D11BD4F33F1DB499 F9A8737BF4FF5C89C903DF31DD78544CF91B1514 9DEAE0DC7063249FB05474681E4AED62986CD25D E463A93F5F3117EEDE6C7316BD02942421F4889F 9D3CC86A72F8494342EA5FD10A41BDC3F4FAFF1C 4DAF18FE948E7A965B30F9457E296D555E7F63A7 28E72909F1717FE9607754F8A7BEB2621678D37D 74E2DEF5D77260B98BC19438099BAD163C70FBFA
```

Comment:
What's the actual criteria for these? It looks like you grabbed ~20 from a list of 57. Is this then set up to only require a single signer of 20 to be considered valid?

Reply:
The only criteria is that they are on the list provided by core. Can add more or less but these are common on other dockerfiles as well.

Reply:
1) Do we only require a single signer?
2) Is the list by core core, or is it a list by core? If its effectively a keyserver than we shouldn't carry this bad practice. It also doesn't explain why the other 37 were excluded.

Reply:
We need to have the key of the signer imported when we verify. These are the most relevant signers for bitcoin. We can adjust this to grab and iterate through https://raw.githubusercontent.com/bitcoin/bitcoin/master/contrib/builder-keys/keys.txt, but it's not necessary and would slow down build times. The practice here is correct, each developer publishes their keys independently to various key servers then the footprints are published across various sources.

Reply:
That doesn't answer either of my questions. I fully understand we need these keys. My questions are:

1. Do we only require a single signer?
2. What were the criteria for these 57 people? With Monero, generally anyone who contributes can be on the list. Their presence is not an endorsement of them. As humans, we need to review to decide who we trust.
3. What were the criteria for the 20 you selected? If we're picking some over others, I'd like to know why. If it's a list of Bitcoin contributors, without strict requirements on inclusion, we should be picking a subset yet we need to discuss that subset.

Reply:
1. Yes, we need to have the key for the single signer imported when we verify afaik bitcoin does not multisign builds at this time.
2 & 3. No particular criteria, these keys were verified to be on the list of core builders and found to be common on multiple other docker files. Bitcoin recommends iterating / importing the list in entirety. If they feel the full list is secure, then our subset only adds to security in this way.